lib/rubocop/cop/style/operator_method_call.rb

```diff
@@ -50,11 +53,10 @@ def method_call_with_parenthesized_arg?(argument)
           argument.children.first && argument.parent.parenthesized?
         end
 
-        def anonymous_forwarding?(argument)
-          return true if argument.forwarded_args_type? || argument.forwarded_restarg_type?
-          return true if argument.hash_type? && argument.children.first&.forwarded_kwrestarg_type?
+        def disallow_argument?(argument)
```

Comment:
Hm, `disallow_argument?` might be a bit ambiguous. What do you think about the names `invalid_syntax_argument?` method and `INVALID_SYNTAX_ARG_TYPES` constant?

```suggestion
        def invalid_syntax_argument?(argument)
```

Or `malformed_argument?` and `MALFORMED_ARG_TYPES`, etc...

Reply:
`invalid_syntax_argument` sounds fine